source/concepts/applications.rst

```diff
@@ -4,99 +4,91 @@ Applications
 ============
 
 The :ref:`NanoVer protocol <base-protocol>` defines how clients and servers
-communicate with each other. It leaves ample room for customisation: it does
+communicate with each other. It is designed to be extensible: it does
 not define the content of a :ref:`FrameData <frame-description>`, the
-meaning of the shared state keys is not defined either, nor does the protocol
-specifies what the commands should be. This points are defined by applications.
-Applications define how to use the different services and how to format the
-content for specific tasks.
+meaning of the shared state keys, or the available commands. Each of these
+can be defined and extended to support various **applications**. Each application
+defines conventions for utilising the different services for specific objectives.
 
 .. _multiplayer-application:
 
-The multiplayer application
+The multi-user application
 ---------------------------
 
-Clients that implement the multiplayer application can send and receive avatars
-so users can share a virtual space. It is the core of multi-user use cases.
+The multi-user application provides a shared virtual space in which users can
+embody themselves with "avatars" and manipulate objects (e.g the simulation box
+in MD).
```

Comment:
What exactly does this mean? "e.g the simulation box in MD." 

Does it mean the scale/position in VR? It's not clear